js/VimeoView.js

```diff
@@ -71,14 +72,27 @@ define([
             type: eventType,
             data: data
           });
-        }.bind(this))
+        }.bind(this));
       }, this);
+
+      if (!this.model._pauseWhenOffScreen) return;
+
+      this.$el.on('inview.pauseWhenOffScreen', this.onPlayerInview.bind(this));
+    },
+
+    onPlayerInview: function(event, isInView) {
```

Comment:
I think the function name could be better

Reply:
any suggestion? I only really used `onPlayerInview` so that it's similar to [the equivalent function](https://github.com/adaptlearning/adapt-contrib-media/blob/288d6e1bdcf65d6f84cfd6f30272eb83cb0bd13c/js/adapt-contrib-media.js#L334-L336) in adapt-contrib-media...